protocols/raft/src/main/java/io/atomix/protocols/raft/storage/log/RaftLog.java

```diff
@@ -51,12 +52,12 @@ public RaftLogWriter writer() {
 
   @Override
   public RaftLogReader openReader(long index) {
-    return openReader(index, RaftLogReader.Mode.ALL);
```

Comment:
As suggested by Sonar: https://jira.sonarsource.com/browse/RSPEC-3252

Basically `RaftLogReader` inherits from `JournalReader`, where the `Mode` enum is actually define, but it was being accessed through the child class. It's clearer to access the enum directly than indirectly, especially since we gain nothing from accessing it indirectly imo.

Reply:
Ok thanks agree looks also weird I thought these both are different